.azure-pipelines/pr_test_scripts.yaml

```diff
@@ -489,6 +489,10 @@ multi-asic-t1-lag:
 
 dpu:
   - dash/test_dash_vnet.py
```

Comment:
Can script `dash/test_dash_vnet.py` really run on kvm or just skip?

Reply:
Ze will work on these dash scripts , but it may need 2-3 months or more, it's not depends on our efforts, so Xin and I decided to skip them with issue

Reply:
What I want to say is, if `dash/test_dash_vnet.py` also skip on kvm, in conditional mark yaml file, we can only use to skip all of them. 
```
dash:
  skip:
    reason: "Currently dash tests are not supported on KVM"
    conditions:
      - "asic_type in ['vs'] and https://github.com/sonic-net/sonic-mgmt/issues/16407"
```